docker/main/requestservice/entrypoint.sh

```diff
@@ -50,4 +50,10 @@ python -m ${SERVICE_PACKAGE_NAME:?} \
     ${_DEBUG_ARG:-} \
     --scheduler-host ${SCHEDULER_ENDPOINT_HOST:?} \
     --scheduler-port ${SCHEDULER_ENDPOINT_PORT:?} \
-    --scheduler-ssl-dir ${SCHEDULER_CLIENT_SSL_DIR:?}
+    --scheduler-ssl-dir ${SCHEDULER_CLIENT_SSL_DIR:?} \
```

Comment:
It might help to add all of these environment variables as possible defaults or 'fall-backs' if they are all required and all read from environment variables. Anything to make it pick-up-and-play. This also feeds into my previous comment of providing a more centralized access point for {service: (port, host, ssl directory)} data.